terravaloper1fvj38kgxhf3hqa8yxrartuguqjd4er5jfmmf7y/profile.json

```diff
@@ -0,0 +1,12 @@
+{
+  "contact": {
+    "website": "https://diamondnodes.io",
+    "email": "hello@diamondnodes.io",
+    "twitter": "@_DiamondNodes",
+    "discord": "Diamond Nodes#3413",
+    "github": "DiamondNodes" ,
```

Comment:
@DiamondNodes Please remove the trailing comma after your Github username and then we can merge.  Thanks!

Reply:
Sorry about that, done!